.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Question
+about: Ask a question about using the software
+title: "[Question] "
+labels: ["question", "to confirm"]
+assignees: ""
+---
+
+Please, feel free to ask your question here (you can also ask your question on [our IRC channel](https://web.libera.chat/?nick=webguest?#exodus-privacy)).
```

Comment:
We could put a link to https://exodus-privacy.eu.org/en/page/who/#contact to make sure it's uptodate instead of a link to IRC

Reply:
Done